chore: version 2.10.2 (#1580)
kyma-project/api-gateway: docs/release-notes/2.10.2.md

## New Features

> - Faster requeue when object has been modified error happens [#1496](https://github.com/kyma-project/api-gateway/pull/1496)

Reviewer: ```suggestion
- We've implemented a faster requeue when the error `object has been modified` occurs [#1496](https://github.com/kyma-project/api-gateway/pull/1496)
```

> - Full support for special characters in asterisk paths [#1569](https://github.com/kyma-project/api-gateway/pull/1569)

Reviewer: ```suggestion
- We've added full support for special characters in asterisk paths [#1569](https://github.com/kyma-project/api-gateway/pull/1569)
```
